editor/scripts/bundle.py

```diff
@@ -48,6 +48,7 @@ def get_tag_suffix(self):
 
 # defold/build_tools
 import run
+import http_cache
```

Comment:
Since the initial "import imp" code, we've added to the sys.path, and can use that standard way of doing it instead.